packages/runner-ct/cypress/component/app.spec.tsx

```diff
@@ -0,0 +1,47 @@
+/* global cy */
```

Comment:
Did you run this spec? It should likely fail in runtime because config.env is missing. And plugins.js file is pointing to the incorrect path to webpack config (.js instead of .ts)

:D we likely need to run this on CI

Reply:
Ahhh yes I built the webpack.config.js manually. I'll update this to run on CI, and fix that problem w.r.t the webpack.config.js/ts.

Reply:
The spec does work with the correct config file :)

Reply:
I am actually working on totally the same spec and added more tests